ftplugin/fish.vim

```diff
@@ -31,8 +31,16 @@ endif
 " argument to fish instead of man.
 execute 'setlocal keywordprg=fish\ '.expand('<sfile>:p:h:h').'/bin/man.fish'
 
+let b:match_ignorecase = 0
+if has('patch-7.3.1037')
+    let s:if = '\%(else\s\+\)\@15<!if'
+else
+    let s:if = '\%(else\s\+\)\@<!if'
+endif
+
 let b:match_words =
-            \ escape('<%(begin|function|if|switch|while|for)>:<end>', '<>%|)')
```

Comment:
Any particular reason you dropped this use of `escape`? I think it's more readable.